### ISSUE TITLE
Accessibility Fixes - Tooltips

### ISSUE THREAD

**Issue by jchow11** [bcgov/performance#488]
### **Description**
Hover-over tooltips can restrict accessibility. Tooltips should be used sparingly when possible, and must be able to receive focus and be activated by keyboard-only users. A toggletip is a type of tooltip that is opened when clicking or tapping on the trigger button. They are persistent, and only close when clicking or tapping outside of the toggletip. We are changing the convention in the PDP app to use toggletips as our standard. 

[Here are some dev guidelines on how to implement toggletips with passing accessibility.](https://ltdesign.azurewebsites.net/pattern-library/messaging/toggletips.html#:~:text=A%20toggletip%20is%20a%20type,it%20(see%20accessibility%20notes).)

- [ ] Remove tooltips from dropdown menus
![Screen Shot 2022-05-30 at 5.15.25 PM.png](https://images.zenhubusercontent.com/61e98ff375d0505e601dd00e/05d80672-3dd2-4ae3-ac99-5a5dac7df155)

- [ ] Remove hover-over tooltips from buttons 
![Screen Shot 2022-06-14 at 10.36.28 AM.png](https://images.zenhubusercontent.com/61e98ff375d0505e601dd00e/b0c51a5a-6bbd-4000-b539-e45dfe07cec7)

- [ ] Toggle tips close upon click of anywhere else in the app, outside of the toggle tip window. Ex. If 1 toggle tip is open, and I click another toggle tip "i" icon, the first toggle tip window closes.

- [ ] Add toggletips as per designs below 

### **My Current Goals Designs**
![My Goals _ Current Goals.jpg](https://images.zenhubusercontent.com/61e98ff375d0505e601dd00e/92e2284f-a43f-40d5-8a4b-5be0eab2c106)
![My Goals _ Current Goals _ Toggletip.jpg](https://images.zenhubusercontent.com/61e98ff375d0505e601dd00e/fbb457b8-41fa-421b-b4b7-e519568d3a75)

**Comment by jchow11** [comment]
@Travis-A-Clark assigning to you for review before assigning Dennis and Ji.

**Comment by Travis-A-Clark** [comment]
@jchow11 I think this looks great. Please share with Dennis and Ji so that they begin to implement across the platform. Thanks.

**Comment by acincotelus** [comment]
HR Admin and Sys Admin tooltips (Goal) - completed
My Goals>Create New Goal tooltips- completed
My Goals > Edit Goal tooltips - completed
My Goals > Dashboard  tooltips - in progress

**Comment by acincotelus** [comment]
Deployed to DEV - tooltips 
My Goals > My Current  Goals

**Comment by Travis-A-Clark** [comment]
@acincotelus Thanks for this! Looking much better. One follow-up:
<br>
There is still one hover-over that won't go away. When I used the "Shared with" drop down on the goal card, a little pop up appears that says "No one". It doesn't disappear no matter who I select or what I do afterward.
<br>
![image.png](https://images.zenhubusercontent.com/5ffe1f472e351dd9b643025b/9b8ec26b-d873-4a47-bceb-439e33f9e3b8)
<br>
2)

**Comment by acincotelus** [comment]
Deployed to DEV
Updates on tooltips for Tags and Shared with:
My Goals>Card (Shared with)
My Goals >Create New Goal (Tags)
My Goals >Edit Goal (Tags)

**Comment by acincotelus** [comment]
updates deployed to Test environment
